vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -822,6 +842,11 @@ def apply(
         apply_router_weight_on_input: bool = False,
         activation: str = "silu",
     ) -> torch.Tensor:
+
+        indices_type = None
+        if layer.use_deepep_kernels or layer.use_deepep_ll_kernels:
+            indices_type = torch.int64
```

Comment:
Similarly, add a method that provides the `indices_type` that we could query at `select_gemm_impl` time.